### ISSUE TITLE
Add back single command target

### ISSUE THREAD

**Issue by Prim69** [df-mc/dragonfly#719]
Many commands require a single target such as things like ban, setrank, mute, etc. Having to treat every command as multiple targets is very inconvenient, this was discussed before and I said I would add it but got busy

**Comment by Sandertv** [comment]
I'm still not entirely convinced these "require" a single target

**Comment by JustTalDevelops** [comment]
I feel like this should be readded also.

I think, while it is easy to 'reproduce' single target functionality yourself, it creates quite a lot of duplicate and unnecessary code versus having some sort of built-in type to handle them - for basically no cost on our part.

**Comment by mintylevel** [comment]
i agree, there should be a way to handle single targets instead of having to always use many. even though you can always ensure that only one target is provided it's very inconvenient and unnecessary.

**Comment by Studgi** [comment]
I agree single target should be a built in feature as well
